python/twitted/twitted/README.md

```diff
@@ -0,0 +1,75 @@
+splunk-sdk-python twitted example
+========================================
+
+This app provides an example of Reporting Custom search commands.
+1. `hashtags` will list out all the hashtags from the tweets
+2. `tophashtags` will list out top _**n**_ hashtags where _**n**_ is passed as an option. _(i.e. top=5)_
+
+### To run this example locally, follow the below steps.
+
+### Step 1
+Execute the following command from the root of this repository.
+```shell
+SPLUNK_VERSION=latest docker compose up -d
+```
+
+### Step 2
+
+Check the container health, run:
+```shell
+docker ps
+```
+
+### Step 3
+
+Make sure STATUS is **healthy** for **splunk-app-examples** container.
+
+Copy the `twitted/twitted` folder in `etc/apps/` inside of the container.
+
+Execute the following command from the root of this directory.
+```shell
+docker cp python/twitted/twitted splunk-app-examples:/opt/splunk/etc/apps/twitted
```

Comment:
I think this needs updating to remove steps 3+

Reply:
Missed this one. Thanks for catching it.
I have updated the README.md, mounted twitted example, and updated README.md for the repository.